dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -26,17 +26,74 @@ class ScriptLevelsControllerTest < ActionController::TestCase
     @custom_s2_l2 = create(:script_level, script: @custom_script,
                            stage: @custom_stage_2, :position => 2)
     client_state.reset
+
+    Gatekeeper.clear
+    DCDO.clear
   end
 
   test 'should show script level for twenty hour' do
     @controller.expects :slog
 
-    get :show, script_id: @script, stage_id: @script_level.stage.position, id: @script_level.position
+    get_show_script_level_page(@script, @script_level)
```

Comment:
while you're at it you could probably simplify this to just take the `@script_level` since `@script` should be the same as `@script_level.script`